parm/config/gefs/config.resources

```diff
@@ -241,6 +241,32 @@ case ${step} in
     export is_exclusive=True
     ;;
 
+  "postsnd")
+    walltime="02:00:00"
+    export ntasks=141
+    export ntasks_postsndcfp=9
+    case "${CASE}" in
+      "C768")
+        tasks_per_node=21
+        threads_per_task=6
+        ;;
+      "C1152")
+        tasks_per_node=9
+        threads_per_task=14
+        ;;
+      *)
+        tasks_per_node=21
+        threads_per_task=6
+        ;;
```

Comment:
You could probably drop the case statement altogether and just have `tasks_per_node=21; threads_per_task=6`.